src/odemis/dataio/tiff.py

```diff
@@ -350,6 +351,7 @@ def _convertToOMEMD(images, multiple_files=False, findex=None, fname=None, uuids
 #        + ARData ({0, 1})
 #        + POLData ({0, 1})
 #        + StreakCamData ({0, 1})
+#        + ExtraSettings   # To store all hw settings
```

Comment:
An alternative to inventing new extension to the OME standard would be to try to fit it to something of OME. I think the most fitting would be https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome_xsd.html#CommentAnnotation .... which could be ok, if using a specific description or namespace. Still, it's probably not that exciting, because it's quite a bit more complex (as you need to use a AnnotationRef). So long story short, I agree with you to create a special attribute ;-)